### DISCUSSION TITLE
Suggestion for "about dialog box" text

### DISCUSSION THREAD

**Discussion by developer1105** [linuxmint/discussions#369] [Ideas]
Just a suggestion for the text on the about dialog...

Instead of:

_Nemo lets you organize files and folders, both on your computer and online._

It could be like this, but centered:

_Organize files and folders
both on your computer and online
with Nemo file manager_

I know, this suggestion might seem useless, but maybe you find it a little beautiful :-)

**Comment by JosephMcc** [comment]
Your ignoring tranlations. It might look nice in English but any of those lines could be much longer in another language and then it just looks weird.

**Comment by developer1105** [comment]
Ok
